app/assets/javascripts/application.js

```diff
@@ -14,21 +14,16 @@
 //= require jquery3
 //= require popper
 //= require bootstrap-sprockets
-//= require bs-custom-file-input.js
-//= require bs-file-input-init.js
-//= require react
-//= require react_ujs
 //= require lodash
-//= require components
 //= require tablesort
 //= require tablesort/dist/sorts/tablesort.number.min
-//= require teleconsultation-fields
-//= require_tree .
+//= require bs-custom-file-input
+//= require_tree ./common
```

Comment:
Include common JS here that can be easily bundled up for the whole site. This allows us to clearly segregate kitchen-sink JS with highly-specialized JS.

Reply:
This also allows us to avoid loading the following for every page:

* React
* Progress Tab JS
* Help screen JS

Reply:
Yeah, we were.